src/openrct2/actions/TrackDesignAction.cpp

```diff
@@ -238,7 +239,11 @@ GameActions::Result TrackDesignAction::Execute() const
 
     auto numCircuits = std::max<uint8_t>(1, _td.operation.numCircuits);
     SetOperatingSettingNested(ride->id, RideSetSetting::NumCircuits, numCircuits, flags);
-    ride->setToDefaultInspectionInterval();
+
+    uint8_t defaultInspectionInterval = Config::Get().general.DefaultInspectionInterval;
+    if (defaultInspectionInterval <= RIDE_INSPECTION_NEVER)
+        SetOperatingSettingNested(ride->id, RideSetSetting::InspectionInterval, defaultInspectionInterval, flags);
```

Comment:
I realized this a bit too late but this will now cause desyncs if other clients have a different value set. The correct way would be to pass this as an argument to the game action.

Reply:
Good point, I created https://github.com/OpenRCT2/OpenRCT2/issues/24126 to keep track of it.

But I’m not sure the desync was _introduced_ here though - the `ride->setToDefaultInspectionInterval()` function, which it called before this PR, seems to have the same issue.